app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -51,6 +53,55 @@ mod css_id {
 
 
 
+// ========================
+// === ClickableElement ===
+// ========================
+
+
+// === ClickClosure ===
+
+/// Type alias for "on-click" event handlers on buttons.
+type ClickClosure = Closure<dyn FnMut(MouseEvent)>;
+
+
+// === ClickableElement ===
+
+/// Clickable HTML element. It has a single `click` event source that fires an event on each `click`
+/// JS event.
+#[derive(Debug, Clone, CloneRef)]
+struct ClickableElement {
```

Comment:
`ClickableElement` is a pretty universal abstraction that can be used in other parts of the project, but I wasn't sure what is the right place for it and decided to leave it here, in WelcomeScreen for now. We can move it somewhere else in a separate task (and update other code, like documentation visualization).